### PR TITLE
only log item on keep of item

### PR DESCRIPTION
**Pull request by egut125**
uiManager now knows game_stats
keep item now logs items pickit

### CODE REVIEW

File: src/item/pickit.py [hunk 1]
```diff
@@ -130,7 +130,6 @@ def pick_up_items(self, char: IChar, is_at_trav: bool = False) -> bool:
                         # send log to discord
                         if found_items and closest_item.name not in picked_up_items:
                             Logger.info(f"Picking up: {closest_item.name} ({closest_item.score*100:.1f}% confidence)")
-                            self._game_stats.log_item_pickup(closest_item.name, self._config.items[closest_item.name].pickit_type == 2)
```
Comment (aeon0) [inline src/item/pickit.py:133]: the pickit doesnt have any more dependency on the game_stats. Can you remove it from the members vars, the constructor and the calls towards it?

**Review by aeon0 — CHANGES_REQUESTED** [review]

File: src/item/pickit.py (hunk 1 above)
Comment (egut125) [reply src/item/pickit.py:133]: Good find

File: src/bot.py [hunk 2]
```diff
@@ -43,7 +43,7 @@ def __init__(self, screen: Screen, game_stats: GameStats, pick_corpse: bool = Fa
         self._config = Config()
         self._template_finder = TemplateFinder(self._screen)
         self._item_finder = ItemFinder(self._config)
-        self._ui_manager = UiManager(self._screen, self._template_finder)
+        self._ui_manager = UiManager(self._screen, self._template_finder, self._game_stats)
         self._belt_manager = BeltManager(self._screen, self._template_finder)
         self._pather = Pather(self._screen, self._template_finder)
         self._pickit = PickIt(self._screen, self._item_finder, self._ui_manager, self._belt_manager, self._game_stats)
```
Comment (aeon0) [inline src/bot.py:49]:
```suggestion
        self._pickit = PickIt(self._screen, self._item_finder, self._ui_manager, self._belt_manager)
```

Comment (egut125) [reply src/bot.py]: ah crap

**Review by aeon0 — APPROVED** [review]

**Pull request merged**